counsel.el

```diff
@@ -3133,6 +3133,11 @@ A is the left hand side, B the right hand side."
   :group 'ivy
   :type 'string)
 
+(defcustom counsel-yank-pop-ivy-height 5
+  "The `ivy-height' of `counsel-yank-pop'."
+  :group 'ivy
+  :type 'number)
```

Comment:
This should be `integer` instead of `number`.

Reply:
Thanks, updated.